lib/engines/content_object_store/app/lib/content_object_store/publishable.rb

```diff
@@ -8,12 +8,18 @@ def publish_with_rollback(schema:, title:, details:)
       ActiveRecord::Base.transaction do
         content_block_edition = yield
         content_id = content_block_edition.document.content_id
+        organisation_id = content_block_edition.lead_organisation.content_id
 
         create_publishing_api_edition(
           content_id:,
           schema_id: schema.id,
           title:,
           details: details.to_h,
+          links: {
```

Comment:
I do wonder whether we're getting to the point of needing a presenter to marshal all this stuff to send to the API. Not a concern for now though